BlackNectar/Carthage/Checkouts/AromaSwiftClient/.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: objective-c
+cache: cocoapods
+rvm: 2.2.2
+osx_image: xcode8
+before_install:
+  - brew update
+
+  # Only if app has Carthage dependencies
+  - carthage bootstrap --verbose
```

Comment:
Check that, nevermind. I see that's just a carthage checkout.

@cordhernandez Please update this so that the `Carthage/Sources`  folder is not included in the source code.